docs/Technical-Documentation/jwt-key-rotation.md

```diff
@@ -0,0 +1,80 @@
+# Rotating JWT Keys
+
+## Context
+
+To maintain good security, we will periodically rotate the JWT keys used to control authentication and authorization to our application. This document outlines the process of how to do this.
+
+**Warning** Production sites will need to be taken down for maintenance when rotating keys, as the rotation will automatically invalidate all current sessions.
+
+### 1. Generate New Keys
+
+In your Mac terminal (or bash terminal in Windows), enter the following command:
+```bash=
+ssh-keygen -t rsa -b 4096 -m PEM -f jwtRS256.key
+```
+You will receive the following response. Do not add a passphrase when prompted.
```

Comment:
see above suggested change that would replace this portion of output if acceptable.

Reply:
I'm not sure which portion this refers to. Are there specific line numbers?

Reply:
15-28 (as you noted below)